src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java

```diff
@@ -39,6 +42,58 @@
     @Override
     public void configure() {
 
+        /**
+         * Get the list of tasks that are already executed, by process definition key
+         *
+         * sample url:
+         * localhost:5000/channel/task/2251799813686414
+         *
+         * sample response: {
+         *   "tasks": [
+         *     ""
+         *   ]
+         * }
+         */
+        from(String.format("rest:get:/channel/task/{%s}", PROCESS_DEFINITION_KEY))
```

Comment:
Please follow a consistent API spec:
/category/{category_id}/sub-category

So this should be:
/process/{processId}/task

Reply:
LGTM